src/Twig/Extension/RecordExtension.php

```diff
@@ -53,6 +53,7 @@ public function getFilters()
             new TwigFilter('selectfield', [Runtime\RecordRuntime::class, 'selectField']),
             new TwigFilter('trimtext',    [Runtime\RecordRuntime::class, 'excerpt'], $safe + $deprecated + ['alternative' => 'excerpt']),
             new TwigFilter('taxonomy',    [Runtime\RecordRuntime::class, 'taxonomy']),
+            new TwigFilter('findimage',   [Runtime\RecordRuntime::class, 'findimage'], $safe + $deprecated),
```

Comment:
It isn't "finding an image" it is "returning the first image"

Reply:
I thought about naming, and this wat the best i came up with. "getImage" would be confusing, as it is too close to the records current magic getter, and "getfirstimage" is a bit long. I don't mind changing it, because it is literally used in one location now.

Reply:
What about just `first_image` (Twig) and `getFirstImage()` (PHP)? Both speak to what it does.

Reply:
Fair enough. 👍